.github/workflows/ci.yml

```diff
@@ -0,0 +1,72 @@
+name: CI Checks
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    name: Build Docker Image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: get short SHA
+        id: sha
+        run: |
+          echo ::set-output name=sha_short::$(git rev-parse --short=7 ${{ github.sha }})
+      - name: Docker meta
+        id: docker_meta
+        uses: crazy-max/ghaction-docker-meta@v1
+        with:
+          images: quay.io/dannymassa/testrepo
+          # Add custom labels that aren't generated by docker-meta action
+          label-custom: |
+            org.opencontainers.image.authors=AT&T Services INC

```

Comment:
We will want these labels in the Dockerfile itself, if it's built outside of github actions, the labels still need to be created.  You can use ARGS to set default labels: https://microbadger.com/labels.